community/blog/2020-04-Releasing-Arcus-Templates-v0.2.0/README.md

```diff
@@ -0,0 +1,46 @@
+# Announcing Arcus Templates
+
+Recently we released v0.2.0 of the [Arcus.Templates](https://github.com/arcus-azure/arcus.templates/releases/tag/v0.2.0) project and is available on NuGet.
+This library will boost development for new projects with option-in/out of commonly used functionality.
+
+Take a look what theses templates have to offer!
+
+## Web API Template
+
+Version v0.1.0 only contained some basic functionality, exception handling, and an option to add a shared key authentication mechanism.
+What version [v0.2.0](https://www.nuget.org/packages/Arcus.Templates.WebApi/) provides is a lot more than that! 
+
+Here's what the default template contains without any additional configurations:
+
+* [Exception middleware](https://webapi.arcus-azure.net/features/logging) to log unhandled exceptions thrown during request processing.
+* Content negotiation that only supports `application/json`.
+* OpenAPI docs generation and UI (only available locally).
+* Provides basic health endpoint with [ASP.NET Core health checks](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/health-checks?view=aspnetcore-2.2) with [OpenAPI support](https://www.codit.eu/blog/documenting-asp-net-core-health-checks-with-openapi/).
+* Docker building file.
+* Default console logger.
+
+Besides that, we have provided a whole range of additional functionality that can be optioned-in/out during the creation of the template:
+
+* We improved the authentication in our API by adding [JWT](https://webapi.arcus-azure.net/features/security/auth/certificate) and [client certificate](https://webapi.arcus-azure.net/features/security/auth/certificate) authentication.
+* The template now allows you to include a `appsettings.json` file in the project.
+* [Correlation](https://webapi.arcus-azure.net/features/correlation) between HTTP requests/responses is by default included in the project but can be option-out.
+* The project includes [ASP.NET OpenAPI docs generation and UI](https://docs.microsoft.com/en-us/aspnet/core/tutorials/getting-started-with-swashbuckle?view=aspnetcore-3.1&tabs=visual-studio) by default but can be excluded.
+* The logging capabilities are also expanded by adding a default console and [Serilog](https://serilog.net/) option.
```

Comment:
This is mentioned above already a bit, we might want to rephrase?
```suggestion
* If you prefer [Serilog](https://serilog.net/) instead of default console, you can opt-in for that.
```